repo/packages/E/elasticsearch/0/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "elasticsearch",
+  "version": "0.1.0",
+  "tags": ["dcos", "mesos", "framework", "elasticsearch", "elastic"],
+  "maintainer": "pnw@trifork.com",
+  "description": "DCOS implementation of the Mesos-Elasticsearch framework",
+  "scm": "https://github.com/mesos/elasticsearch.git",
+  "website": "https://github.com/mesos/elasticsearch",
+  "images": {
+    "icon-large": "https://www.elastic.co/static/img/logo-elastic.png"
+  },
+  "postInstallNotes": "Elasticsearch is staging. Check Marathon and the ES management front-end for status."
```

Comment:
It would be great to have preInstallNotes as well that indicate the framework needs...
example: https://github.com/mesosphere/universe/blob/version-1.x/repo/packages/C/cassandra/0/package.json#L14-L16